### ISSUE TITLE
Unicode text gets escaped in JSON layout

### ISSUE THREAD

**Issue by biiiipy** [NLog/NLog#2062]
**Type**: Bug
**NLog version**: 4.5.0-alpha04
**Platform**: .Net 4.5

Unicode characters get encoded in JSON layout:
https://github.com/NLog/NLog/blob/master/src/NLog/Targets/DefaultJsonSerializer.cs#L274

This block is not necessary (at least in a UTF8 encoded file target), cause we already escaped control characters in switch statement:
```
if (EscapeChar(ch))
{
     sb.AppendFormat(CultureInfo.InvariantCulture, "\\u{0:x4}", (int)ch);
}
```

My setup:
```
JsonLayout jsonLayout = new JsonLayout()
{
     SuppressSpaces = true
};
jsonLayout.Attributes.Add(new JsonAttribute("msg", @"${message}"));
```

Logging Copyright © symbol: 
``Log.Fatal("©");``
Log output: 
``{"msg":"\u00a9"}``
Expected output: 
``{"msg":"©"}``

Q1: Is there any workaround?
Q2: What's the status with using JSON.Net for serializing?

**Comment by 304NotModified** [comment]
thanks for the bug report!

**Comment by 304NotModified** [comment]
This will be fixed in 4.4.7 and soon also in 4.5 alpha 5

**Comment by 304NotModified** [comment]
fixed in beta 4.5
